creator-node/src/middlewares.js

```diff
@@ -863,10 +878,17 @@ async function getReplicaSetSpIDs({
 
     let errorMsg = null
     try {
-      replicaSet =
-        await libs.contracts.UserReplicaSetManagerClient.getUserReplicaSet(
-          userId
-        )
+      const encodedUserId = libs.Utils.encodeHashId(userId)
+      const spResponse = await libs.discoveryProvider.getUserReplicaSet({
+        encodedUserId
+      })
```

Comment:
doesn't matter to me but would it make sense to move the userId encoding into libs e.g.
```
libs.discoveryProvider.getUserReplicaSet({ encodedUserId, userId }) {
  if (userId) encodedUserId = libs.Utils.encodeHashId(userId)
  ...
}
```